src/app/game/game.component.ts

```diff
@@ -1,23 +1,38 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, HostListener, OnInit, OnDestroy } from '@angular/core';
+import { Subscription } from 'rxjs';
 
-import { Answer } from '../models/answer';
-import { AnswerService } from '../services/answer.service';
+import { GamePlayService } from '../services/game-play.service';
 
 @Component({
   selector: 'app-game',
   templateUrl: './game.component.html',
   styleUrls: ['./game.component.scss']
 })
-export class GameComponent implements OnInit {
-  protected answer: Answer = { id: 0, word: '' };
-  constructor(private answerService: AnswerService) { }
+export class GameComponent implements OnInit, OnDestroy {
+  constructor(private gameplayService: GamePlayService) { }
```

Comment:
Minor nit-pick: it's good practice to make services in the constructor readonly as well:

```typescript
constructor(private readonly gameplayService: GamePlayService) {}
```

Reply:
I've wondered about that one. Just tried and found out that if it's not readonly then you can redefine functions on the object. Wild stuff